README.md

```diff
@@ -0,0 +1,84 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
```

Comment:
- [ ] According to the [Microverse readme template](https://github.com/microverseinc/readme-template), your readme file is missing an "Acknowledgements" section.  Kindly add it so we can meet this particular requirement.

That way, your project will be well documented 💪